compiler/literate/html.ml

```diff
@@ -179,7 +179,10 @@ let rec law_structure_to_html
     let block_content = Marked.unmark c in
     check_exceeding_lines start_line filename block_content;
     Format.fprintf fmt
-      "<div class='code-wrapper%s'>\n<div class='filename'>%s</div>\n%s\n</div>"
+      "<div class='code-wrapper%s catala-code'>\n\
```

Comment:
Yeah I should have done that a long time ago. This will make https://github.com/CatalaLang/catala-website/blob/23d4993b10120746407774ac4716249808bc2c8f/src/components/CatalaCode.res#L20 unnecessary